src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -655,6 +667,69 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
             }
         }
 
+        /// <summary>
+        /// Creates a dictionary which maps a request method name to its clr method name via <see cref="JsonRpcMethodAttribute" /> value.
+        /// </summary>
+        /// <param name="target">Object to reflect over and analyze its methods.</param>
+        /// <returns>Dictionary which maps a request method name to its clr method name.</returns>
+        private void GetRequestMethodToClrMethodMap(object target)
```

Comment:
This method no longer returns anything. Perhaps `Get` isn't the right verb to lead its name with.